gui/default/index.html

```diff
@@ -344,40 +344,99 @@ <h3 class="panel-title">
+              <div class="panel-body">
+                <p><span translate>Passkeys are a modern alternative to passwords that is both more secure and often easier to use. Set it up in GUI settings to use it.</span> <a href="{{docsURL('users/webauthn')}}" translate>Learn more</a></p>
+                <p class="text-info" ng-if="!isLocationInsecure() && !webauthnAvailable()" translate>Your browser does not support WebAuthn.</p>
+                <div ng-if="locationDoesNotMatchWebauthnRpId()">
+                  <p class="text-info" translate>Current domain does not match WebAuthn configuration.</p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>Syncthing Documentation: Customizing the GUI address</a>
+                  </p>
+                </div>
+                <p class="text-info" ng-if="isLocationInsecure()" translate>WebAuthn requires HTTPS on non-localhost domains.</p>
+                <p class="text-danger" ng-if="webauthn.errors.notRegistered" translate>This device or security key is not registered. Please try a different one.</p>
+                <p class="text-info" ng-if="webauthn.errors.aborted" translate>Authentication aborted by user.</p>
+                <p class="text-danger" ng-if="webauthn.errors.notAllowed" translate>Authentication failed, please try again.</p>
+                <div ng-if="webauthn.errors.securityError">
+                  <p class="text-danger" translate>Initialization blocked by browser. Please make sure Syncthing is correctly configured.</p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>Syncthing Documentation: Customizing the GUI address</a>
+                  </p>
+                </div>
+                <p class="text-danger" ng-if="webauthn.errors.initFailed" translate>Initialization failed, see Syncthing logs for details.</p>
+                <p class="text-danger" ng-if="webauthn.errors.authenticationFailed" translate>Authentication failed, see Syncthing logs for details.</p>
+                <p class="text-info" ng-if="webauthn.errors.noCredentials" translate>No WebAuthn credentials configured.</p>
+                <p class="text-danger" ng-if="webauthn.errors.uvRequired" translate>User verification is required, but was not performed.</p>
+              </div>
+              <div class="panel-footer clearfix">
+                <label class="stay-logged-in">
+                  <input type="checkbox" ng-model="login.stayLoggedIn" ng-disabled="login.inProgress">&nbsp;<span translate>Stay logged in</span>
+                </label>
+
+                <button ng-if="isLocationInsecure() || locationDoesNotMatchWebauthnRpId()" type="button" class="btn btn-primary" ng-click="reloadLoginAtWebauthnAddress()" translate translate-value-webauthn-address="{{ inferWebauthnAddress() }}">Reload page at {%webauthnAddress%}</button>
```

Comment:
```suggestion
                <button ng-if="isLocationInsecure() || locationDoesNotMatchWebauthnRpId()" type="button" class="btn btn-primary" ng-click="reloadLoginAtWebauthnAddress()" translate translate-value-address="{{ inferWebauthnAddress() }}">Reload page at {%address%}</button>
```

Reply:
This translation string is used in 2 places, so both need to be updated in tandem (and might as well rename the same variable in other translation strings too). Fixed in f081706a319e1006d05f6eefd9fb3f7278cc5eb3.